Makefile

```diff
@@ -107,7 +112,11 @@ pytest-valgrind: all
 
 endif
 
+ifdef WITH_GCOV
+test: all pytest
```

Comment:
nit: tab

Reply:
this is a rule, I don't think we want it indented

Reply:
Aah yes